src/test/java/com/wikia/webdriver/common/core/api/GraphApi.java

```diff
@@ -0,0 +1,127 @@
+package com.wikia.webdriver.common.core.api;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.entity.UrlEncodedFormEntity;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+import org.openqa.selenium.WebDriverException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * Created by neptunooo on 04/08/16.
+ */
+public class GraphApi {
+
+    private static String ERROR_MESSAGE = "Problem with Graph API call used to create new facebook test user";
+    private static ArrayList<BasicNameValuePair> PARAMS;
+    private static String wikia_production_app_access_token = "112328095453510|uncl945c48bixtm47AWZM64doDQ";
+    private static String wikia_preview_app_access_token = "983287518357559|RE3IANMJOSmv04gSFO7hThMWVBI";
+
+    public HashMap<String, String> createFacebookTestUser(String app_id) {
+        try {
+            HttpResponse response = createTestUser(app_id);
+            InputStream body = response.getEntity().getContent();
+            String entity = EntityUtils.toString(response.getEntity());
+            HashMap<String, String> result =
+                new Gson().fromJson(entity, new TypeToken<HashMap<String, String>>(){}.getType());
+            return result;
+        }
+        catch (IOException e) {
+            PageObjectLogging.log("URI_SYNTAX EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+            throw new WebDriverException(ERROR_MESSAGE);
+        }
+        catch (URISyntaxException e) {
+            PageObjectLogging.log("URI_SYNTAX EXCEPTION", ExceptionUtils.getStackTrace(e), false);
+            throw new WebDriverException(ERROR_MESSAGE);
+        }
+
+    }
+
+    public HashMap<String, String> deleteFacebookTestUser(String app_id, String user_id) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00117)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The return type of this method should be an interface such as "Map" rather than the implementation "HashMap". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1319)